### DISCUSSION TITLE
Unable To Access Settings - stuck at password prompt

### DISCUSSION THREAD

**Discussion by ValleyForge2** [mcguirepr89/BirdNET-Pi#1265] [Q&A] [answered]
Hello - my first install of BirdNET on a Pi5 using the Pi5 download script. Install went well.
When I go to "Tools" to edit the settings, I keep getting the password prompt over and over again.
Others have reported this: I have checked the birdnet.conf latitude and longitude values which are correct. I've even deleted them and it's no better.
Another issue is illegal characters in passwords - only letters and numbers.
Any help?

**Comment by stevedee** [comment, marked as answer]
This is said to be due to recent updates to PHP: https://github.com/Nachtzuster/BirdNET-Pi/issues/243

The **_McGuire_** version of BirdNet-Pi is now well out of date, so I suggest you start again with the **_Nachtzuster_** version which is very actively maintained: https://github.com/Nachtzuster/BirdNET-Pi

**Reply by ValleyForge2** [reply]
Thank you - once you scratch the surface and find these things....
All the information i had found to date was pointing to the McGuire version.
